src/theme/css/chrome.css

```
@@ -252,7 +252,7 @@ pre > .buttons button {
         padding: 0.3rem 1rem;
     }
 }
-code {
+p code {
```

This doesn't look quite right to me. This padding should only be applied to code blocks which have the form:

```
<pre><code>…</code></pre>
```

`p code` would apply to almost everything *except* code blocks.

It seems to me that the rule should be something like:

```suggestion
pre > code {
```

(which also matches the other nearby rules.)

---

Ah, sorry I misunderstood the original PR. I've changed it to your suggestion, and amended it to the old commit since it's small.